packages/server/src/services/challenge.service.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { ChallengeState } from '../enums';
+import { Achievement, Challenge } from '../models';
+import { startNewChallenge } from './challenge.service';
+import { achievementsMock } from './__mocks__/achievement.mock';
+import { challengesMock } from './__mocks__/challenge.mock';
+import { tasksMock } from './__mocks__/task.mock';
+
+const mapToAchievementId = (achievement: Achievement): string => achievement.id;
+
+describe('challenge service', () => {
+  describe('startNewChallenge', () => {
+    let newChallenge: Challenge;
```

Comment:
it`s not good idea to create let and use it in all cases, lets move it to beforeEach